apps/kaizen/models.py

```
@@ -1,3 +1,56 @@
+        return self.title
+
+
+# Reflection model
+class Reflection(BaseModel):
+    RATING_CHOICES = [
+        (1, "Very Bad"),
+        (2, "Bad"),
+        (3, "Neutral"),
+        (4, "Good"),
+        (5, "Very Good"),
+    ]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)